<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattlefieldChatClass</class>
 <widget class="QMainWindow" name="BattlefieldChatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>417</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Battlefield 1 中文输入工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="BattlefieldChat.qrc">
    <normaloff>:/BattlefieldChat/icon1.ico</normaloff>:/BattlefieldChat/icon1.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 250, 250);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>200</y>
      <width>657</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QWidget" name="mainContent" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>16</y>
      <width>673</width>
      <height>185</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="chkAllowBypassLimit">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>0</y>
       <width>273</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>绕过游戏聊天长度限制 (可能会带来额外风险)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkSupportFullscreen">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>0</y>
       <width>185</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>实验性全屏支持 (按Home呼出)</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>24</y>
       <width>673</width>
       <height>161</height>
      </rect>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>预处理模式</string>
      </attribute>
      <widget class="QGroupBox" name="groupPreprocessMode">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>8</y>
         <width>649</width>
         <height>105</height>
        </rect>
       </property>
       <property name="title">
        <string>预处理模式</string>
       </property>
       <widget class="QRadioButton" name="radioModeNop">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>24</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>不处理</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioModeTrad">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>48</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>转换为繁体</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioModeEnglish">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>72</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>翻译为英文</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>104</x>
          <y>24</y>
          <width>537</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(162, 162, 162);</string>
        </property>
        <property name="text">
         <string>不执行任何预处理 (不推荐: 简体中文无法在游戏中显示)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>104</x>
          <y>48</y>
          <width>537</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(162, 162, 162);</string>
        </property>
        <property name="text">
         <string>将简体中文转换为繁体中文 (默认)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>104</x>
          <y>72</y>
          <width>537</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(162, 162, 162);</string>
        </property>
        <property name="text">
         <string>使用在线服务翻译为英文 (发送前需要额外等待翻译完成)</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>在线翻译设置</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>8</y>
         <width>257</width>
         <height>129</height>
        </rect>
       </property>
       <property name="title">
        <string>提供商</string>
       </property>
       <widget class="QRadioButton" name="radioProviderDeepL">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>24</y>
          <width>233</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>DeepL (翻译质量最好 国内连接速度慢)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioProviderBaidu">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>48</y>
          <width>233</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>百度翻译 (需要自行申请密钥)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>272</x>
         <y>8</y>
         <width>185</width>
         <height>68</height>
        </rect>
       </property>
       <property name="title">
        <string>百度翻译API密钥</string>
       </property>
       <widget class="QLineEdit" name="editTranslateBaiduKey">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>16</y>
          <width>33</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>APPID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="editTranslateBaiduAppid">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>16</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>40</y>
          <width>33</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>密钥</string>
        </property>
       </widget>
      </widget>
      <widget class="QCheckBox" name="chkTranslateKeepOriginal">
       <property name="geometry">
        <rect>
         <x>464</x>
         <y>32</y>
         <width>201</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>翻译后保留中文原文</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>464</x>
         <y>8</y>
         <width>54</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>翻译超时</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="editTranslateTimeout">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>8</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>500</number>
       </property>
       <property name="maximum">
        <number>30000</number>
       </property>
       <property name="singleStep">
        <number>500</number>
       </property>
       <property name="value">
        <number>5000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>648</x>
         <y>8</y>
         <width>17</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lblCurrentStatus">
    <property name="geometry">
     <rect>
      <x>504</x>
      <y>8</y>
      <width>177</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>正在等待游戏启动</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listLogs">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>216</y>
      <width>673</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(31, 31, 31);
color: rgb(255, 255, 255);
font: 9pt &quot;Consolas&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BattlefieldChat.qrc"/>
 </resources>
 <connections/>
</ui>
